config.yml

```diff
@@ -18,6 +18,8 @@ transcription_default:
   vad_filter: True
   condition_on_previous_text: False
 
+translation_default_method: "segmented" # "segmented" | "full"
```

Comment:
does this need quotation marks? I think string is the default type for yaml in this case, no?